src/components/ImageDisplay/ThumbnailRow.vue

```diff
@@ -1,22 +1,37 @@
 <template>
-  <div class="images">
-    <img
-      v-for="(item, index) in images"
-      :key="index"
-      :src="thumbnailWithFallback(item)"
-      onerror="this.onerror=null;this.src='https://via.placeholder.com/60/FF0000/FFFFFF?text=jpg'"
-      :title="item.base_filename"
-      :class="{'selected_thumbnail' : item.image_id == selected_image}"
-      loading="lazy"
-      class="recent-image"
-      @click="setActiveImage(item)"
+  <div
```

Comment:
Do we need these extra divs? Can we design our loop a bit like what was there before? I notice you're only showing the parent div if there are grouped images - AFAIK the old code still shows the image carousel with the "no jpg available" even when there aren't images - I think we want to replicate that

Reply:
fixed it!